site/src/pages/docs/server-timeouts.mdx

```diff
@@ -0,0 +1,210 @@
+# Overriding server timeouts
+
+You can override the default timeouts of a request.
```

Comment:
Since `request timeout` isn't the only type of timeout on server-side (connection timeouts, etc..)

```suggestion
There are several ways to override the default timeouts at server-side.
```